contents/teams/replay/objectives.mdx

```diff
@@ -1,17 +1,24 @@
-### 🎯 Goal 1: 10 Happy Android Replay Users 
-
-We'll also release an alpha of iOS Mobile Replay.
+### 🎯 Goal 1: Quality and Consolidation for Session Replay (web & mobile)
 
 Why? 
 
-We've done the work's base and now we want to release the new Android SDK with session replay and onboard customers to validate the product. 
+We've done a lot of investigation and preparation work and we want to capitalize on that
+
+* Mobile replay out of beta for Android/iOS
+* Build the best UX for capturing and playing recordings
```

Comment:
I wonder if this fully captures what we want to achieve from a capture / playback point of view. Maybe something like:
```suggestion
* Make capture and playback as complete, performant and easy as possible
```